docs/pallas/grid_blockspec.md

```diff
@@ -107,7 +103,10 @@ to get the actual element index on the corresponding array axis.
 If the block shape does not divide evenly the overall shape then the
 last iteration on each axis will still receive references to blocks
 of `block_shape` but the elements that are out-of-bounds are padded
-on input and discarded on output. Note that at least one of the
+on input and discarded on output. The padding is unspecified, and
```

Comment:
"The values in the padding are unspecified" to be a little more concrete? "padding is unspecified" sounds like the amount of padding is undetermined which is not true